### PR TITLE
#2875 BaseClasses.mo, BaseClasses_INIT.mo and other .mo split into in…

### LINKED ISSUE #2875: Split the BaseClasses.mo models into several models
The following BaseClasses.mo are to be splitted : 

Dynawo/Electrical/Controls/Transformers/BaseClasses.mo
Dynawo/Electrical/Controls/Voltage/BaseClasses.mo
Dynawo/Electrical/HVDC/BaseClasses.mo
Dynawo/Electrical/Loads/BaseClasses.mo
Dynawo/Electrical/Machines/BaseClasses.mo
Dynawo/Electrical/Machines/OmegaRef/BaseClasses.mo
Dynawo/Electrical/Machines/SignalN/BaseClasses.mo
Dynawo/Electrical/StaticVarCompensators/BaseClasses.mo
Dynawo/Electrical/Transformers/BaseClasses.mo

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes #2875

### CODE REVIEW

**Comment by cossartque** [comment]
This branch has conflicts, can you solve them and rebase so that I can review ? Thanks

**Review by cossartque — COMMENTED** [review]
Can you confirm that these are just changes in the BaseClasses organization ?

**Comment by ErwanGuichard49** [comment]
The following .mo files have been split :

Dynawo/AdditionalIcons/package.mo
Dynawo/Electrical/Controls/Basics/SwitchOff.mo
Dynawo/Electrical/Controls/Transformers/BaseClasses.mo
Dynawo/Electrical/Controls/Transformers/BaseClasses_INIT.mo
Dynawo/Electrical/Controls/Voltage/BaseClasses.mo
Dynawo/Electrical/Controls/WECC/Parameters.mo
Dynawo/Electrical/Events/Event.mo
Dynawo/Electrical/HVDC/BaseClasses.mo
Dynawo/Electrical/HVDC/BaseClasses_INIT.mo
Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/Parameters.mo
Dynawo/Electrical/Loads/BaseClasses.mo
Dynawo/Electrical/Machines/BaseClasses.mo
Dynawo/Electrical/Machines/BaseClasses_INIT.mo
Dynawo/Electrical/Machines/OmegaRef/BaseClasses.mo
Dynawo/Electrical/Machines/OmegaRef/BaseClasses_INIT.mo
Dynawo/Electrical/Machines/SignalN/BaseClasses.mo
Dynawo/Electrical/Machines/SignalN/BaseClasses_INIT.mo
Dynawo/Electrical/StaticVarCompensators/BaseClasses.mo
Dynawo/Electrical/StaticVarCompensators/BaseControls/Parameters.mo
Dynawo/Electrical/Transformers/BaseClasses.mo
Dynawo/Electrical/Transformers/BaseClasses_INIT.mo

**Comment by ErwanGuichard49** [comment]
Moreover, I have applied the camel-case convention to several model names : for instance

BaseTapChangerPhaseShifter_MAX --> BaseTapChangerPhaseShifterMax
Params_DriveTrain --> ParamsDriveTrain

In Dynawo/Electrical/HVDC/HvdcPTanPhi/HvdcPTanPhi.mo, there is an unexplained modification : Q1Pu and Q2Pu have been replaced by QInj1Pu and QInj2Pu. I will reverse it.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/GeneratorSynchronousExtTfo_3E_INIT.mo [hunk 1]
```diff
@@ -18,7 +18,7 @@ model GeneratorSynchronousExtTfo_3E_INIT "Synchronous machine with 3 windings -
   import Dynawo.Electrical.Machines;
 
   extends Machines.BaseClasses_INIT.BaseGeneratorVariables_INIT;
-  extends BaseClasses_INIT.BaseGeneratorSynchronousExt3E_INIT;
+  extends BaseClasses_INIT.BaseGeneratorSynchronousExt_3E_INIT;
```
Comment (ErwanGuichard49) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/GeneratorSynchronousExtTfo_3E_INIT.mo:21]: I have renamed BaseGeneratorSynchronousExt3E_INIT as BaseGeneratorSynchronousExt_3E_INIT, this change is unwarranted but inconsequential (even on dynawo-rte) : a future PR will rename it BaseGeneratorSynchronousExt3W_INIT.

**Review by cossartque — APPROVED** [review]

**Pull request merged**